src/.eslintrc

```diff
@@ -8,7 +8,7 @@
   "globals": {
     "__DEV__": false,
     "__VERSION__": false,
-    "__CONFIG__": false
+    "__BUILD_CONFIG__": false
```

Comment:
The type of `__BUILD_CONFIG__` is BuildConfigType. This could cause problems. Also it is not expected that `__BUILD_CONFIG__` is null or false.

Reply:
Maybe convert this file to a .js file and load some default.

Reply:
Do you have an example for a js eslintrc file?

Reply:
https://eslint.org/docs/user-guide/configuring#specifying-globals

Changed it to `readonly`